api/handlers/process_test.go

```diff
@@ -454,4 +460,79 @@ var _ = Describe("Process", func() {
 			})
 		})
 	})
+
+	Describe("DELETE /v3/processes/:guid/instances/:instance", func() {
+		When("the instance and process are found", func() {
+			BeforeEach(func() {
+				processRepo.GetProcessReturns(repositories.ProcessRecord{
+					GUID:             "process-guid",
+					AppGUID:          "app-guid",
+					SpaceGUID:        "space-guid",
+					DesiredInstances: 2,
+					Type:             "web",
+				}, nil)
+
+				appRepo.GetAppReturns(repositories.AppRecord{
+					GUID:      "app-guid",
+					SpaceGUID: "space-guid",
+					Revision:  "revision",
+				}, nil)
+
+			})
+
+			JustBeforeEach(func() {
```

Comment:
We could pull the `JustBeforeEach` up and parametrize the instance number to avoid repeating essentially the same `JustBeforeEach` in every `When`